PopupTabLayout/PopupTabLayout.js

```diff
@@ -350,6 +353,29 @@ PopupTabLayout.Tab = Tab;
  * @ui
  */
 
+const tabPanelsHandlers = {
+	onTransition: handle(
+		forward('onTransition'),
+		(ev, props, {onTransition}) => {
+			onTransition(ev);
+		}
+	),
+	onKeyDown: handle(
+		forward('onKeyDown'),
+		forKey('left'),
+		(ev, {index}) => (index > 0),
+		(ev) => {
+			if (Spotlight.move('left') || getContainerNode(getContainersForNode(ev.target).pop()).tagName === 'HEADER') {
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/117918613-515b5000-b326-11eb-9d39-a8df43b02a55.png)


Please check the qa-sampler FixedPopupPanels > with various item
And I am worried about the `Slider` behavior

Reply:
👏 Nice catch! I updated the logic to escape this and added `Slider` component to the qa sampler.